sources/models/defaults.toml

```
@@ -52,6 +52,24 @@ template-path = "/usr/share/templates/updog-toml"
 affected-services = ["updog"]
 seed.setting-generator = "bork seed"
 
+# Healthdog
+
+[settings.healthdog]
+# the URL to which anonymous health metrics will be sent
+metrics-url = "https://metrics.bottlerocket.aws/metrics"
```

```suggestion
metrics-url = "https://metrics.bottlerocket.aws"
```
We don't need this, do we?

---

Addressed in [f18d841](https://github.com/bottlerocket-os/bottlerocket/compare/ec0abbf943fe3806294f53fd4e118be3d1f529fc..f18d841ee1c2b35d5f09136ea3f7136e2a4d65ae)